source/features/collapse-markdown-sections.ts

```diff
@@ -0,0 +1,48 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function toggleSection(sectionHeading: HTMLElement): void {
+	const sectionState = sectionHeading.classList.contains('rgh-markdown-section-collapsed');
+	sectionHeading.classList.toggle('rgh-markdown-section-collapsed');
+
+	let element = sectionHeading.nextElementSibling as HTMLElement;
+	while (element) {
+		if (element.matches('h1, h2, h3, h4, h5, h6')) {
+			if (sectionHeading.tagName.localeCompare(element.tagName) >= 0) {
```

Comment:
```suggestion
			if (sectionHeading.tagName[1] >= element.tagName[1]) {
```

I’ve arrived at micro optimizations 😂